components/gripper/server.go

```diff
@@ -66,3 +66,16 @@ func (s *subtypeServer) Stop(ctx context.Context, req *pb.StopRequest) (*pb.Stop
 	}
 	return &pb.StopResponse{}, gripper.Stop(ctx)
 }
+
+// IsMoving queries of a component is in motion.
+func (s *subtypeServer) IsMoving(ctx context.Context, req *pb.IsMovingRequest) (*pb.IsMovingResponse, error) {
```

Comment:
did we miss adding all the IsMoving methods before? @cheukt

Reply:
they were all added. we originally made isMoving an explicitly local method but I don't think we can keep it as is under this scheme because then there is no way for the "parent" robot to query for whether the modular resource is moving or not.

Reply:
Correct, it was local-only (and thus not in the grpc server/client) and I moved it to be a full/normal method. I think it has to be like this, because it can be safety critical (so can't just skip/workaround), and everything modular is on the other side of a grpc call.